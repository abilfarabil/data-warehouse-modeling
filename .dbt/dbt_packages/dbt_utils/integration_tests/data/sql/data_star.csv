version https://git-lfs.github.com/spec/v1
oid sha256:ecafbb6703406f567aa01cc348fb15f134c0b2536f5dbd07b5a6ca58f5c69b7a
size 46
