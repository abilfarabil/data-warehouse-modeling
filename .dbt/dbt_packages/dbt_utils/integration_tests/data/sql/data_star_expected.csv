version https://git-lfs.github.com/spec/v1
oid sha256:93ce5fbc27ab55840ea926a626ca45498452e900df7d374487e204e8c1373120
size 32
